### PR TITLE
[SMALLFIX] Improve Swift documentation

### PR DESCRIPTION
**Pull request by madanadit**

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-Swift.md [hunk 1]
```diff
@@ -85,9 +83,9 @@ To stop Alluxio, you can run:
 
 {% include Common-Commands/stop-alluxio.md %}
 
-## Running functional test with IBM SoftLayer
+# Running functional tests
 
-Configure your Swift or SoftLayer account in the `tests/pom.xml`, where `authMethodKey` should be
+Configure your Swift account credetntials under `swiftTest` in `tests/pom.xml`, where `authMethodKey` should be
```
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-Swift.md]: credentials

File: docs/en/Configuring-Alluxio-with-Swift.md [hunk 2]
```diff
@@ -57,12 +56,11 @@ On the successful authentication, Keystone will return two access URLs: public a
...
 value of `<swift-use-public>`  to `false`.
 
+## Options for Swift Object Storage
 
-### Accessing IBM SoftLayer object store
-
-Using the Swift module also makes the IBM SoftLayer object store an option as an under storage
-system for Alluxio.  SoftLayer requires `<swift-auth-model>` to be configured as `swiftauth`
-
+Using the Swift module makes Ceph Object Storage, using RadosGW, and IBM SoftLayer Object Storage as under storage
```
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-Swift.md]: I don't quite understand this sentence ... is `RadosGW` a module for `Ceph` ?

**Comment by madanadit** [comment]
@ifcharming PTAL. Thanks!.

File: docs/en/Configuring-Alluxio-with-Swift.md [hunk 3]
```diff
@@ -57,12 +56,11 @@ On the successful authentication, Keystone will return two access URLs: public a
...
 
+## Options for Swift Object Storage
 
-### Accessing IBM SoftLayer object store
-
-Using the Swift module also makes the IBM SoftLayer object store an option as an under storage
-system for Alluxio.  SoftLayer requires `<swift-auth-model>` to be configured as `swiftauth`
-
+Using the Swift module makes Ceph Object Storage and IBM SoftLayer Object Storage as under storage
+options for Alluxio. To use Ceph, the Rados Gateway module must be deployed.
```
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-Swift.md]: nit : can you please provide links to Ceph, SoftLayer and Rados ?

**Comment by madanadit** [comment]
@jsimsa Can you merge this? Thanks.

**Pull request merged**